dlt/extract/incremental/transform.py

```diff
@@ -117,18 +120,49 @@ def find_cursor_value(self, row: TDataItem) -> Any:
 
         Will use compiled JSONPath if present, otherwise it reverts to column search if row is dict
         """
-        row_value: Any = None
         if self._compiled_cursor_path:
-            row_values = find_values(self._compiled_cursor_path, row)
-            if row_values:
-                row_value = row_values[0]
+            cursor_values = find_values(self._compiled_cursor_path, row)
+            if cursor_values == []:
+                if self.on_cursor_value_missing == "raise":
```

Comment:
I'm sure you can refactor this function that this code:
```
if self.on_cursor_value_missing == "raise":
                    raise IncrementalCursorPathMissing(self.resource_name, self.cursor_path, row)
                elif self.on_cursor_value_missing == "exclude":
                    return None
```
will be in it only once. right now it is duplicated 3 times

Reply:
Thank you for the suggestion. I'm not sure how to achieve this because:
1. the exception in the case of "raise" is different. We have `IncrementalCursorPathMissing` and `IncrementalCursorPathHasValueNone`
2. we want to return None from this function in case the user wants to exclude such rows. I don't know how to return from the `find_cursor_value()` function if I extract the check for "exclude" and then `return None` into a separate function.

Reply:
Thanks for stepping in and doing the refactoring!